app/controllers/event.js

```diff
@@ -0,0 +1,143 @@
+const Event = require('../models/Event')
+module.exports = {
+  createEvent: async (req, res, next) => {
+    const event = new Event(req.body)
+    try {
+      await event.save()
+      res.status(201).json({ event: event })
+    } catch (error) {
+      res.status(400).json({ error: 'Unable to create the event!' })
+    }
+  },
+  updateEvent: async (req, res) => {
+    Event.findById(req.params.id, (err, data) => {
+      if (err) {
+        res.status(400).json({ error: 'invalid update' })
```

Comment:
why would this be an `invalid update?`. 
api did not find the event, thats what this case is hence thrown proper error.